templates/firefox-ios.xliff

```diff
@@ -2433,6 +2551,22 @@
         <source>Studies</source>
         <note>Label used as a toggle item in Settings. When this is off, the user is opting out of all studies.</note>
       </trans-unit>
+      <trans-unit id="Settings.Tabs.CustomizeTabsSection.InactiveTabs" xml:space="preserve">
+        <source>Inactive Tabs</source>
+        <note>Option in settings to toggle Inactive Tabs in the settings menu</note>
+      </trans-unit>
+      <trans-unit id="Settings.Tabs.CustomizeTabsSection.TabGroups" xml:space="preserve">
+        <source>Tab Groups</source>
+        <note>Option in settings to toggle Tab Groups in the settings menu</note>
```

Comment:
Please give brief description of what a Tab Group is for more context on how to translate it best